operantanalysis/operantanalysis.py

```diff
@@ -95,7 +93,16 @@ def respondingduringcueanditi(timecode, eventcode, codeon, codeoff):
                 itipoke += 1
         itipokerpm = itipoke / (cuelengthsec / 60)
         allpokeitirpm += [itipokerpm]
-    print(allpokeitirpm)
-    print(allpokerpm)
     
     return round(statistics.mean(allpokerpm), 3), round(statistics.mean(allpokeitirpm), 3)
+
+
+def leverpressing(eventcode, lever1, lever2=False):
```

Comment:
docstring(s) please describing input/output(s)

Reply:
Just added documentation for this function in my new branch